tests/unit/s2n_release_non_empty_buffers_test.c

```diff
@@ -59,6 +59,7 @@ int mock_client(struct s2n_test_io_pair *io_pair)
     s2n_connection_free(conn);
     s2n_config_free(client_config);
     s2n_cleanup();
+    EXPECT_SUCCESS(s2n_io_pair_close_one_end(io_pair, S2N_CLIENT));
```

Comment:
You shouldn't be calling any s2n functions after s2n_cleanup

Reply:
I have moved `s2n_io_pair_close_one_end` before `s2n_cleanup`.